editor/editor_node.cpp

```diff
@@ -1278,6 +1278,15 @@ void EditorNode::save_resource_in_path(const Ref<Resource> &p_resource, const St
 }
 
 void EditorNode::save_resource(const Ref<Resource> &p_resource) {
+	// If built-in script, save the scene instead.
```

Comment:
```suggestion
	// If built-in resource, save the scene instead.
```
Now that it's universal.